.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ on:  # yamllint disable-line rule:truthy
     branches: ['master', 'v1-10-test', 'v1-10-stable', 'v2-0-test']
 
 env:
-
+  AIRFLOW_COMMITERS: ${{ secrets.AIRFLOW_COMMITERS }}
```

Comment:
I guess we can remove this one? That turned out to be not working for PRs I believe?